### PR TITLE
Fixed path for clean target

### LINKED ISSUE #48: `make clean` has the wrong paths
## Current behaviour
`make clean` tries to delete statefiles etc. inside examples, which has none, so nothing is deleted

## Expected behaviour
Delete tmp-files inside example

## How to reproduce?
Try running `make clean`

## Suggestion(s)/solution(s) [Optional]
Change path to delete inside `examples/standalone/` instead of `examples`

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by pdmthorsrud**
## Closes/fixes/resolves issue(s)?
closes #48 

## What was added/changed/fixed?
changed paths in recipe for `clean` target

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [ ] Added to milestone

### CODE REVIEW

**Review by dangernil — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 - Add intentions section in readme #42
 
+### Changed
+
+- changed paths in `make install` to delete tmp-files in all subdirectories #48
+
```
Comment (dangernil) [inline CHANGELOG.md:12]:
```suggestion
### Changed
- changed paths in `make install` to delete tmp-files in all subdirectories #48

```

File: Makefile [hunk 2]
```diff
@@ -53,9 +53,9 @@ remove-tmp:
 	rm -rf ./tmp
 	rm -rf ./.vagrant
 	rm -rf ./.minio.sys
-	rm -rf ./example/.terraform
-	rm -rf ./example/terraform.tfstate
-	rm -rf ./example/terraform.tfstate.backup
+	rm -rf ./example/**/.terraform
+	rm -rf ./example/**/terraform.tfstate
+	rm -rf ./example/**/terraform.tfstate.backup
```
Comment (claesgill) [inline Makefile:58]: Why two asterisks? 🤔

Comment (pdmthorsrud) [reply Makefile:58]: Goes through all subdirs https://stackoverflow.com/questions/8532929/two-asterisks-in-file-path

**Review by zhenik — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (pdmthorsrud) [reply CHANGELOG.md:12]: Just following what's written from before :)

File: Makefile (hunk 2 above)
Comment (claesgill) [reply Makefile:58]: Nice!

**Pull request merged**